README.md

```diff
@@ -67,6 +67,29 @@ Note that testing is done on specific minor versions, but any minor versions wit
   - Self-managed (on-premises)
   - Managed cloud offerings (e.g., Amazon OpenSearch, Amazon OpenSearch Serverless)
 
+### Performance Limitations
+A performance test was performed on 3/10/25 alongside [PR 1337](https://github.com/opensearch-project/opensearch-migrations/pull/1337)
+to identify general indexing throughput with Reindex-From-Snapshot and Traffic Replay. While Reindex-From-Snapshot
+includes periods of ingestion inactivity (e.g. while downloading the shard data), this was not factored into the ingestion rates.
+Test used docs with an average uncompressed size 2.39 KB (source doc and index command) which corresponded to 1.54 KB of primary shard size per doc on OS 2.17 with default settings
+
+For real-word use, throughput can be increased by vertically scaling Reindex-From-Snapshot and Traffic Replay instances or horizontally scaling
+the Reindex-From-Snapshot workers that are running on AWS Fargate. Outside the exception indicated, all cases are CPU limited for throughput.
+All cases were using uncompressed network traffic generated by the applications, results will vary if using client compression.
+
+Throughput here is measured by the rate of increase in primary shard data with bulk ingestion on the target cluster alongside
+the uncompressed size of the source data ingested.
+
+#### Test Results
+| Service               | vCPU | Memory (GB) | Type Mapping Sanitization | Peak Docs Ingested per minute | Primary Shard Data Ingestion Rate (MBps) | Uncompressed Source Data Ingestion Rate (MBps) |
```

Comment:
Users may not understand what this mean. I'd add a brief description or a reference.